src/services/iiif-parser.js

```diff
@@ -112,13 +121,11 @@ export function getTracks() {
  */
 export function getLabelValue(label) {
   if (label && typeof label === 'object') {
-    // English
-    if (label.hasOwnProperty('en')) {
-      return label['en'].length > 0 ? label['en'][0] : '';
-    }
-    // None
-    if (label.hasOwnProperty('none')) {
-      return label['none'].length > 0 ? label['none'][0] : '';
+    const labelKeys = Object.keys(label);
+    if (labelKeys && labelKeys.length > 0) {
+      // Get the first key's first value
+      const firstKey = labelKeys[0];
+      return label[firstKey].length > 0 ? label[firstKey][0] : '';
```

Comment:
:+1: I think this is good to not assume a particular language code.  If we want to provide the user a choice of language in the future I think there are some patterns for dealing with these language maps in UV (or manifest or manifold).